src/solver/DiffusionTurbSolver.h

```diff
@@ -7,6 +7,7 @@
 #ifndef ARTSS_SOLVER_DIFFUSIONTURBSOLVER_H_
 #define ARTSS_SOLVER_DIFFUSIONTURBSOLVER_H_
 
+#include <string>
```

Comment:
intentional?

Reply:
yes DiffusionSolver uses std::string so it should be included, as a header needs to be self contained.

Reply:
sure, but aren't includes supposed to be where they are used? In this case the include has to be in the cpp file and not the header file?

Reply:
done in db36360